### DISCUSSION TITLE
[Gantt] Ressources footer is messed when in monthly mode

### DISCUSSION THREAD

**Discussion by jtplaarj** [plantuml/plantuml#1972] [Q&A]
Hi,

When in monthly mode, the resources are printed weekly and it is quite messed:

![monthly with resources](https://github.com/user-attachments/assets/0d5192e5-5133-4b45-8ef8-ca8c046ea114)

Is there any way to make the resources also monthly?

**Comment by The-Lum** [comment]
_See evolution on:_
- #1973
